src/DotVVM.Framework/Utils/ReflectionUtils.cs

```diff
@@ -226,8 +226,18 @@ public static object ConvertValue(object value, Type type)
             // convert
             return Convert.ChangeType(value, type, CultureInfo.InvariantCulture);
         }
-
+        private static ConcurrentDictionary<string, Type> cache_FindTypeHash = new ConcurrentDictionary<string, Type>();
+        private static ConcurrentDictionary<string, Type> cache_FindTypeHashIgnoreCase = new ConcurrentDictionary<string, Type>();
```

Comment:
We can pass StringComparer.OrdinalIgnoreCase as a comparer to the dictionary - Roslyn uses this comparer for symbol equality, so it should be safe. http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/SymbolKey/SymbolKey.SymbolKeyComparer.cs,21

Reply:
Fixed